controllers/k8ssandra/k8ssandracluster_controller_test.go

```diff
@@ -297,6 +298,13 @@ func createSingleDcCluster(t *testing.T, ctx context.Context, f *framework.Frame
 		}
 		return false
 	}, timeout, interval)
+
+	// Check that the datacenter has the original cluster name, without sanitization.
+	// dc1 := cassdcapi.CassandraDatacenter{}
+	// err = f.Get(ctx, dcKey, &dc1)
+	// require.NoError(err, "failed to get CassandraDatacenter")
+	// require.Equal(kc.Spec.Cassandra.ClusterName, dc1.Spec.ClusterName)
```

Comment:
Commented out?

Reply:
Or just dead code and the later test is correct?

Reply:
🤦 
I commented it out while I was fixing some other related bugs.
Definitely need to get this uncommented to do the actual check on the cluster name that is passed to the cassdc object.